flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -674,6 +674,14 @@
 			.outputTypeStrategy(TypeStrategies.MISSING)
 			.build();
 
+	// json functions
+	public static final BuiltInFunctionDefinition JSON_EXISTS =
```

Comment:
we don't need to add it here if we only aim to SQL.

Reply:
> we don't need to add it here if we only aim to SQL.

well, I removed it.